src/common.hpp

```diff
@@ -7,7 +7,7 @@
 
 #define XXH_STATIC_LINKING_ONLY
 #define XXH_PRIVATE_API
-#define XXH_ACCEPT_NULL_INPUT_POINTER
+#define XXH_ACCEPT_NULL_INPUT_POINTER 1
```

Comment:
This is the only change in this commit that isn't strictly copy and pasting files.